tests/unit/s2n_session_ticket_test.c

```diff
@@ -1348,11 +1348,56 @@ int main(int argc, char **argv)
         EXPECT_SUCCESS(s2n_config_free(config));
     }
 
+    /* Session key decryption with TLS 1.3 using decrypt-only session key */
```

Comment:
I think a better description of this test is "Test: TLS1.3 resumption is successful when key used to encrypt ticket is in decrypt-only state"